spec/features/user_can_log_in_and_out_spec.rb

```diff
@@ -4,14 +4,14 @@
     scenario "logs into users account" do
       user = create(:user)
 
-      visit "/"
+      visit root_path
```

Comment:
@jtruong2 @erose357 @bretfunk What do we think about using the root path helper vs declaring the specific url we want to be displayed? I see value in both. 

I think we can use helpers for directing the user visits and then actually write out the expected url in our expectations.